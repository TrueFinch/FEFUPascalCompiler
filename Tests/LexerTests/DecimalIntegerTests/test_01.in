9223372036854775807
0
1337
10
100
1000
10000
100000
1000000
10000000
100000000
1000000000
10000000000
100000000000
1000000000000
10000000000000
100000000000000
1000000000000000
10000000000000000
100000000000000000
1000000000000000000
1 2 3 4 5 6 7 8 9
10 11 12 13 14 15 16 17 18 19
20 21 22 23 24 25 26 27 28 29
30 31 32 33 34 35 36 37 38 39
40 41 42 43 44 45 46 47 48 49