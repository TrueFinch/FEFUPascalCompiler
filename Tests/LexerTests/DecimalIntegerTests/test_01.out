1,1 TYPE_INTEGER 9223372036854775807 9223372036854775807
2,1 TYPE_INTEGER 0 0
3,1 TYPE_INTEGER 1337 1337
4,1 TYPE_INTEGER 10 10
5,1 TYPE_INTEGER 100 100
6,1 TYPE_INTEGER 1000 1000
7,1 TYPE_INTEGER 10000 10000
8,1 TYPE_INTEGER 100000 100000
9,1 TYPE_INTEGER 1000000 1000000
10,1 TYPE_INTEGER 10000000 10000000
11,1 TYPE_INTEGER 100000000 100000000
12,1 TYPE_INTEGER 1000000000 1000000000
13,1 TYPE_INTEGER 10000000000 10000000000
14,1 TYPE_INTEGER 100000000000 100000000000
15,1 TYPE_INTEGER 1000000000000 1000000000000
16,1 TYPE_INTEGER 10000000000000 10000000000000
17,1 TYPE_INTEGER 100000000000000 100000000000000
18,1 TYPE_INTEGER 1000000000000000 1000000000000000
19,1 TYPE_INTEGER 10000000000000000 10000000000000000
20,1 TYPE_INTEGER 100000000000000000 100000000000000000
21,1 TYPE_INTEGER 1000000000000000000 1000000000000000000
22,1 TYPE_INTEGER 1 1
22,3 TYPE_INTEGER 2 2
22,5 TYPE_INTEGER 3 3
22,7 TYPE_INTEGER 4 4
22,9 TYPE_INTEGER 5 5
22,11 TYPE_INTEGER 6 6
22,13 TYPE_INTEGER 7 7
22,15 TYPE_INTEGER 8 8
22,17 TYPE_INTEGER 9 9
23,1 TYPE_INTEGER 10 10
23,4 TYPE_INTEGER 11 11
23,7 TYPE_INTEGER 12 12
23,10 TYPE_INTEGER 13 13
23,13 TYPE_INTEGER 14 14
23,16 TYPE_INTEGER 15 15
23,19 TYPE_INTEGER 16 16
23,22 TYPE_INTEGER 17 17
23,25 TYPE_INTEGER 18 18
23,28 TYPE_INTEGER 19 19
24,1 TYPE_INTEGER 20 20
24,4 TYPE_INTEGER 21 21
24,7 TYPE_INTEGER 22 22
24,10 TYPE_INTEGER 23 23
24,13 TYPE_INTEGER 24 24
24,16 TYPE_INTEGER 25 25
24,19 TYPE_INTEGER 26 26
24,22 TYPE_INTEGER 27 27
24,25 TYPE_INTEGER 28 28
24,28 TYPE_INTEGER 29 29
25,1 TYPE_INTEGER 30 30
25,4 TYPE_INTEGER 31 31
25,7 TYPE_INTEGER 32 32
25,10 TYPE_INTEGER 33 33
25,13 TYPE_INTEGER 34 34
25,16 TYPE_INTEGER 35 35
25,19 TYPE_INTEGER 36 36
25,22 TYPE_INTEGER 37 37
25,25 TYPE_INTEGER 38 38
25,28 TYPE_INTEGER 39 39
26,1 TYPE_INTEGER 40 40
26,4 TYPE_INTEGER 41 41
26,7 TYPE_INTEGER 42 42
26,10 TYPE_INTEGER 43 43
26,13 TYPE_INTEGER 44 44
26,16 TYPE_INTEGER 45 45
26,19 TYPE_INTEGER 46 46
26,22 TYPE_INTEGER 47 47
26,25 TYPE_INTEGER 48 48
26,28 TYPE_INTEGER 49 49